health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/PlanInbox.js

```
@@ -511,13 +594,13 @@
   });
 
   useEffect(() => {
-    if (processData && processData.some((instance) => instance.action === "APPROVE_ESTIMATIONS")) {
+    if (processData && processData.some((instance) => instance.action === config?.disableInstanceAction)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)